unit_tests/sources/declarative/schema/test_dynamic_schema_loader.py

```diff
@@ -64,6 +64,21 @@
                     },
                     "paginator": {"type": "NoPagination"},
                 },
+                "schema_transformations": [
+                    {
+                        "type": "AddFields",
+                        "fields": [
+                            {
+                                "type": "AddedFieldDefinition",
+                                "path": ["StaticField"],
+                                "value": "{{ {'type': ['null', 'string']} }}",
```

Comment:
nit: It does not seem a good UX for the dev to have to set up the value as an object as opposed to:
```
                    {
                        "type": <a type>,
                        "fields": [
                            {
                                "path": StaticField,
                                "value": string,
                            }
                        ],
                    },
```

Why I flagged this as nit is it feels like we could always improve the UX be adding a new `AddFieldsSchema` transformation that would allow to do that, right? I just want to validate that adding this would work and be backward compatible so we're not closing any door in the future

Reply:
Yes, we can define a more schema transformation-oriented interface in the AddFieldsSchema transformation. So I think we ok here.